### PR TITLE
task one

### PR DESCRIPTION
**Pull request by Revanth1998**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -2,7 +2,11 @@ import 'dart:ui';
 
 import 'package:flutter/material.dart';
 
-void main() => runApp(MaterialApp(home: Home()));
+void main() => runApp(MaterialApp(
+      home: Home(),
+      //theme: ThemeData(fontFamily: 'DancingScript'),
```
Comment (boymustafa) [inline lib/main.dart:7]: remove dead code please

File: lib/main.dart [hunk 2]
```diff
@@ -11,38 +15,55 @@ class Home extends StatelessWidget {
...
-                  '1:04 PM',
-                  style: TextStyle(color: Colors.white, fontSize: 32),
-                ),
-                Text(
-                  'Starts in 02:29:46',
-                  style: TextStyle(color: Colors.white, fontSize: 14),
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Text(
```
Comment (boymustafa) [inline lib/main.dart:34]: I tot we decided during our last discussion to create method and then call that method on the TextStyle?

File: pubspec.yaml [hunk 3]
```diff
@@ -63,6 +63,7 @@ flutter:
     - family: DancingScript
       fonts:
         - asset: fonts/DancingScript-VariableFont_wght.ttf
+
```
Comment (boymustafa) [inline pubspec.yaml:66]: Remove dead code please

**Review by boymustafa — COMMENTED** [review]
I would suggest putting all images in one folder in the assets folder because the assets folder not only for images, you might want to put other files like JSON file, etc in the assets folder.

File: pubspec.yaml (hunk 3 above)
Comment (Revanth1998) [reply pubspec.yaml:66]: Okk

**Pull request merged**